tests/common.py

```diff
@@ -0,0 +1,39 @@
+# This file is part of the github2jira project
```

Comment:
`common` is not a valid name for a module. Please try and see if you can name it with something that hints about the content.

Reply:
`testdata.py` ?